### PR TITLE
Change help message

### PR DESCRIPTION
**Pull request by sunjc826**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -160,11 +160,10 @@ Format: `exit`
...
 
 PropertyWhiz's data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
-**TODO**: Change `addressbook.json` filename
 
 ### Editing the data file
 
-PropertyWhiz's data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+PropertyWhiz's data are saved as a JSON file `[JAR file location]/data/propertywhiz.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
```
Comment (trxe) [inline docs/UserGuide.md:168]: I think the section "Archiving data files" (further down) should be removed since it's only part of the original AB3.

**Review by trxe — COMMENTED** [review]
(editing)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -185,14 +184,14 @@ _Details coming soon ..._
...
-Action | Format, Examples
---------|------------------
-**Add** | `add n/PROPERTY_NAME a/PROPERTY_ADDRESS s/SELLER_NAME p/SELLER_PHONE $/PRICE_MIN [t/TAG]…​` <br> e.g., `add n/Blk 123 a/123, Clementi Rd, #04-20, 1234665 s/James Lee sp/61234567 $/100000 t/HDB t/3rm`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/PROPERTY_NAME] [a/PROPERTY_ADDRESS] [s/SELLER_NAME] [p/SELLER_PHONE] [$/PRICE_MIN] [t/TAG]…​`<br> e.g.,`edit 2 s/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Exit** | `exit`
-**Help** | `help`
```
Comment (trxe) [inline docs/UserGuide.md:197]: Is there a need for this part to be reformatted? Otherwise I think we can keep the original formatting.

**Review by trxe — CHANGES_REQUESTED** [review]
Sorry for the double review, but I had forgotten to request changes in the previous review, and I have some comments on the markdown formatting for the Command Summary

File: docs/UserGuide.md (hunk 2 above)
Comment (sunjc826) [reply docs/UserGuide.md:197]: Looks like my vscode autoformatter aligns table columns on save

**Review by trxe — APPROVED** [review]
LGTM

**Pull request merged**